appendices/filters.xml

```diff
@@ -0,0 +1,705 @@
+?>
+]]>
+    </programlisting>
+   </example>
+  </section>
+
+  <section xml:id="filters.convert.iconv">
+   <title>convert.iconv.*</title>
+   <simpara>
+    I filtri <literal>convert.iconv.*</literal> sono disponibili se
+    il supporto <link linkend="book.iconv">iconv</link> � abilitato, e il 
+    loro uso � equivalente a processare tutti i dati dello stream con <function>iconv</function>.
+    Questi filtri non supportano parametri, ma si aspettano che la codifica dell'input
+    e dell'outputoutput siano passate come una parte del nome del filtro, ad esempio 
+    entrambi come 
+    <literal>convert.iconv.&lt;input-encoding&gt;.&lt;output-encoding&gt;</literal> o 
+    <literal>convert.iconv.&lt;input-encoding&gt;/&lt;output-encoding&gt;</literal>
+    (entrambe le annotazioni sono semanticamente equivalenti).
+   </simpara>
+
+   <example>
+    <title>convert.iconv.*</title>
+    <programlisting role="php">
+<![CDATA[
+<?php
+$fp = fopen('php://output', 'w');
+stream_filter_append($fp, 'convert.iconv.utf-16le.utf-8');
+fwrite($fp, "T\0h\0i\0s\0 \0i\0s\0 \0a\0 \0t\0e\0s\0t\0.\0\n\0");
+fclose($fp);
+/* Outputs: This is a test. */
+?>
+]]>
+    </programlisting>
+   </example>
+  </section>
+ </section>
+
+ <section xml:id="filters.compression">
+  <title>Compression Filters</title>
+
+  <simpara>
+   Mentre i <link linkend="wrappers.compression">Compression Wrappers</link>
+   fanno in modo di generare files 
+   gzip e bz2 compatibili con il filesystem locale, non permettono 
+   una generica decompressione sui network streams, e non permettono neppure, 
+   a partire da uno stream non compresso, di passare a uno compresso.  
+   Per questo uso un compression filter pu� essere applicato ad ogni stream ogni volta che si vuole.
+  </simpara>
+
+  <note>
+   <simpara>
+    I compression filters <emphasis>not</emphasis> generano headers e chiusure
+    usate da utilit� a riga di comando come <literal>gzip</literal>. Esclusivamente
+    comprimono e decomprimono le porzioni caricate dei dati dello stream.
+   </simpara>
+  </note>
+
+  <section xml:id="filters.compression.zlib">
+   <title>zlib.deflate and zlib.inflate</title>
+   <simpara>
+    <literal>zlib.deflate</literal> (compressione) e
+    <literal>zlib.inflate</literal> (decompressione) sono implementazioni
+    dei metodi di  compressione decritti in <link xlink:href="&url.rfc;1951">RFC 1951</link>.
+    Il filtro <literal>deflate</literal> parte da tre parametri passati come
+    un array associativo.  
+
+    <parameter>level</parameter> l'intensit� della compressione
+    in una scala da 1 a 9. I numeri pi� alti generano pacchetti pi� piccoli
+    al costo di un pi� lungo processo di elaborazione. Esistono due
+    livelli speciali di compressione: 0 (per non eseguire nessuna compressione), 
+    e -1 (valore interno di default per zlib), 6 � un valore predefinito.
+
+    <parameter>window</parameter> � il logaritmo in base -2 del ciclo di compressione della finestra?.
```

Comment:
Da qui alla riga 311 il discorso è poco chiaro, secondo me andrebbe rivisto anche in inglese